build/Repositories.props

```diff
@@ -1,11 +1,16 @@
 <Project>
-  <ItemDefinitionGroup>
-    <Repository>
-      <Branch>rel/2.0.0</Branch>
-    </Repository>
-  </ItemDefinitionGroup>
-
   <ItemGroup>
+    <!-- Repos being patched -->
+    <Repository Include="EntityFramework" Branch="rel/2.0.1" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
```

Comment:
Just a minor thing and it should be working regardless. Should this (and may be other references) be `EntityFrameworkCore` now since we renamed repo?

Reply:
Good catch. Yeah, looks like dev is still "EF" not "EFCore". Can you make a PR to update it? Also you'll need to update TC  VCS roots so commithashs can still be found.

https://github.com/aspnet/Universe/blob/425c1b83e65953b6dc3bdcbf89e2f4d295fbf855/build/repo.targets#L262-L269